utilities/EluvioCli.js

```diff
@@ -731,16 +849,16 @@ yargs(hideBin(process.argv))
   )
 
   .command(
-    "account_set_tenant <tenant_admins>",
-    "Sets the tenant admins group address for this account.",
+    "account_set_tenant <tenantId>",
```

Comment:
We typically use `tenant_id` but in this case we should just say `tenant` since it's simpler as CLI documentation.